lib/src/components/avatars/avatar.dart

```diff
@@ -66,6 +72,9 @@ class ZetaAvatar extends ZetaStatelessWidget {
     this.upperBadge,
     this.borderColor,
     this.backgroundColor,
+    this.semanticValue = 'avatar',
```

Comment:
can we use label instead of value for these names?

Reply:
yes, they are meant to be label. Blame copilot